app/src/main/java/net/skyscanner/backpack/demo/stories/SubStory.kt

```diff
@@ -36,14 +37,17 @@ open class SubStory : Story() {
   ): View? {
     val stories = arguments?.getStringArray(STORIES) ?: savedInstanceState?.getStringArray(STORIES)
     if (stories != null) {
-      val view = inflater.inflate(R.layout.component_list, container, false)
```

Comment:
I think the only usage left of `component_list` is in the `ComponentRegistry` as an empty placeholder for the "sneak peek" item. It might be good to actually remove the view tab for that, so we can remove the layout and then we can also delete `component_list_content`